src/univariates.jl

```diff
@@ -37,6 +37,10 @@ end
 @compat support{D<:ContinuousUnivariateDistribution}(d::Union{D,Type{D}}) = RealInterval(minimum(d), maximum(d))
 @compat support{D<:DiscreteUnivariateDistribution}(d::Union{D,Type{D}}) = round(Int, minimum(d)):round(Int, maximum(d))
 
+# Type used for dispatch on finite support
+# T = true or false
+immutable FiniteSupport{T} end
```

Comment:
No big deal. We can also tag as it is if it help for other packages because any changes along the lines discussed here wouldn't affect the API.